scanpy/tools/_leiden.py

```diff
@@ -141,18 +141,30 @@ def leiden(
         categories=natsorted(np.unique(groups).astype('U')),
     )
     # store information on the clustering parameters
-    adata.uns['leiden'] = {}
-    adata.uns['leiden']['params'] = dict(
+    adata.uns[key_added] = {}
```

Comment:
Since this is a behavior change, be sure to mention it in the release note.

Reply:
@gokceneraslan could you please address this and rebase the PR?